app/templates/manage/placement/form.hbs

```diff
@@ -28,6 +28,30 @@
       {{/type-ahead}}
     </div>
   </div>
+  <div class="col">
```

Comment:
Can you add a text helper to the template field as well, so the spacing between template + pct and the label doesn't look as weird?